### ISSUE TITLE
Dewall should capture #Seg L

### ISSUE THREAD

**Issue by vpicaver** [jedwards1211/dewalls#14]
CaveWhere cannot tolerate a #Seg L argument which just tells the Walls program to not to count that vector in the total for cave length, ie excludes redundant shots.  If Cavewhere also lets you flag redundant shots this should be easy to fix.

This might be a CaveWhere implementation issue.

**Comment by jedwards1211** [comment]
huh do you have an example of a complete data line?  it might be that I coded up `#S` and `#Segment` but not `#Seg`
Also I think Walls lets the user choose which segment indicates excluded shots?  I don't think it has any fixed way of indicating excluded shots, so an importer would have to let the user pick which segment means excluded.

**Comment by jedwards1211** [comment]
Yeah I'm still looking for the documentation on excluding stuff but this hints at `L` not being a reserved segment:

> Some flag-type attributes may not be mutually exclusive. For example, you may want to use "P" for "don't display" and "L" for "length-exclude." Since a vector could have both attributes, it might be convenient to define a
third attribute, say "PL" to allow more flexibility in distinguishing on a map the three different conditions.

**Comment by jedwards1211** [comment]
In this screen shot the selected segment is actually ` Actun Kaua - North Maze/North Maze Surveys/1994 Additions/A Survey/Upper Level/L` and to exclude it you have to select it and click `Detach`

Kind of a mess...I guess we would need a UI that shows the segment tree for the imported data and allows the user to pick which segments are excluded.

![image](https://user-images.githubusercontent.com/1448194/85767166-5c40ce00-b6dd-11ea-8ec9-720cb723be42.png)

**Comment by jedwards1211** [comment]
I guess we could also give them the option to say any segment ending in L (or whatever convention they're using) is excluded

**Comment by vaxheadroom** [comment]
5-years later, I decide to chime in:   Walls does not have the ability to exclude length based on the content of the SRV file.   "Detaching" segments by name, per the example in the Walls manual, is a work-around.  Not particularly effective, imo.    

Note that in that example in Andy's screenshot, if one clicks "apply to all", then all "L" segments get detached, regardless of their location in tree.   But "detaching" usually will affect other geometry, hence not producing the desired effect, since "length" total will be likely be botched a bit.

"L" and "P" examples in the Walls manual come from Compass flags.  The two other Compass flags are "C" (exclude shot from closure) and "X" (total exclusion).  Walls converts those Compass flags to #Segments when importing compass files, hence their they are commonly seen.   But alas, these special segment names, on their own, do not result in special Walls behavior.

If Breakout were to include an option to use "L", for example, to exclude length, then probably best approach would be an option to specially handle those four characters, when used as segment names in Walls as if they were Compass flags.

JoeWalko
